packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -95,226 +96,232 @@ const getTitle = (market_type: string, show_eu_related_content: boolean) => {
+                                        {localize('Download')}
+                                    </Text>
+                                </a>
+                            </div>
+                            <div className='cfd-trade-modal__download-center-app--option'>
+                                <Icon icon='IcLinuxLogo' size={32} />
+                                <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                                    {localize('MetaTrader 5 Linux app')}
+                                </Text>
+                                <a
+                                    className='dc-btn cfd-trade-modal__download-center-app--option-link'
+                                    type='button'
+                                    href={getPlatformMt5DownloadLink('linux')}
+                                    target='_blank'
+                                    rel='noopener noreferrer'
+                                >
+                                    <Text size='xxs' weight='bold' color='prominent'>
+                                        {localize('Learn more')}
+                                    </Text>
+                                </a>
+                            </div>
+                        </React.Fragment>
+                    )}
+                </div>
+                <Text
+                    align='center'
+                    as='div'
+                    className='cfd-trade-modal__download-center-text'
+                    size={isMobile() ? 'xxxs' : 'xxs'}
+                    weight='bold'
+                >
+                    {localize(
+                        'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
+                        {
+                            platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
+                            account: getAccountTitle(),
+                        }
+                    )}
+                </Text>
+                <div className='cfd-trade-modal__download-center-options'>
+                    <div className='cfd-trade-modal__download-center-options--mobile-links'>
+                        <a href={getPlatformMt5DownloadLink('ios')} target='_blank' rel='noopener noreferrer'>
+                            <Icon icon='IcInstallationApple' width={135} height={40} />
+                        </a>
+                        <a href={getPlatformMt5DownloadLink('android')} target='_blank' rel='noopener noreferrer'>
+                            <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                        </a>
+                        <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
+                            <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                        </a>
+                    </div>
+                    {!isMobile() && (
```

Comment:
Let's make the css changes to hide this part as well. we are using a mix of isMobile and is_mobile while we're aiming to hide the mentioned parts by css.

Reply:
ok